_projects/README.md

```diff
@@ -6,9 +6,11 @@ In the `_projects` folder, create a markdown file for the project. The frontmatt
 - `custom-css: project`
 - `title`
 - `desc` -- a couple of short lines to be rendered in the project card.
-- `github` -- optional -- link to GitHub repo containing the project.
 - `image` -- with a path to the project picture, which you should place in the `/img` folder.
+- `github` -- optional -- link to GitHub repo containing the project.
 - `youtube` -- optional -- Video ID for YouTube video about the project. Video ID is the `v=` query parameter in a YouTube URL. 
 For example, for the video `https://www.youtube.com/watch?v=dQw4w9WgXcQ`, the video ID is `dQw4w9WgXcQ`.
+- `link` -- optional -- Link to the project site.
+- `dl` -- optional -- Link to project download.
```

Comment:
descarga de [????] es una programa ya compilado y empieza Unity Player, o el `.unitypackage` del proyecto, o ...?

Reply:
TBD, I requested that the link be to a `.zip` containing the exe